README.md

```diff
@@ -2,43 +2,43 @@
 
-A **complete list** of the 47 included Metrics, Statistical Techniques and Data Processing Tools is: [available here](https://scores.readthedocs.io/en/latest/included.html).
+A **complete list** of the 47 included metrics, statistical tests and data processing tools is: [available here](https://scores.readthedocs.io/en/develop/included.html).
 
-Here is a **curated selection** of the Metrics and Tools included in `scores`:
+Here is a **curated selection** of the metrics and statistical tests included in `scores`:
 
-| continuous                      | probability | categorical      | statistical tests |
+| Continuous                      | Probability | Categorical      | Statistical tests |
 | ----------                      | ----------- | -----------      | ----------------- |
-| MAE, MSE, RMSE, Flip-Flop Index, Quantile Score, Murphy score, Pearson's Correlation Coefficient, Additive Bias, Multiplicative Bias  | CRPS for CDF, CRPS for ensemble, ROC, Brier Score, Isotonic Regression (reliability diagrams)   | FIRM, Hit Rate, Probability of False Detection, Success Ratio, Peirce's Skill Score  |  Diebold Mariano (with the Harvey et al. 1997 and the Hering and Genton 2011 modifications) |
+| MAE, MSE, RMSE, Additive Bias, Multiplicative Bias, Pearson's Correlation Coefficient, Flip-Flop Index, Quantile Score, Murphy score  | Brier Score, CRPS for CDF, CRPS for ensemble, ROC, Isotonic Regression (reliability diagrams)   | Hit Rate, Probability of False Detection, Success Ratio, Peirce's Skill Score, FIRM  |  Diebold Mariano |
 
 **Notice -- This repository is currently undergoing initial construction and maintenance. It is getting much closer to our goals for version one, but there are a few more things to add. This notice will be removed after the first feature release. In the meantime, please feel free to look around, and don't hesitate to get in touch with any questions (see the contributing guide for how).**
 
 Documentation is hosted at [scores.readthedocs.io](https://scores.readthedocs.io).  
 Source code is hosted at [github.com/nci/scores](https://github.com/nci/scores).  
 The tutorial gallery is hosted at [as part of the documentation, here](https://scores.readthedocs.io/en/latest/tutorials/Explanation.html).
 
-`scores` is a Python package containing mathematical functions for the verification, evaluation, and optimisation of forecasts, predictions or models. It primarily supports the geoscience and earth system science communities. It also has wide potential application in machine learning, and in domains other than meteorology, geoscience and weather.
+`scores` is a Python package containing mathematical functions for the verification, evaluation, and optimisation of predictions and models. It primarily supports the Earth system science community. It also has wide potential application in machine learning and other domains.
```

Comment:
(1) I would consider keeping the word "forecasts" (as in "optimisation of forecasts, predictions or models). 

I wonder if it might help with search engine optimisation? For instance, if people are googling "forecast verification metrics" or something along those lines, maybe including the term "forecasts" will help search engines find `scores`?  

(2) Similarly, I would consider keeping the references to "meteorology, geoscience and weather." Purely for search engine optimisation. 

I guess an alternative second sentence could be something along the lines of "In addition to meteorology, weather and geoscience domains, `scores` has wide potential application in machine learning and other domains"?

Reply:
There's the option to mention "forecasts" elsewhere. For instance, we have
`About the Scores Project` as the header. I suggest that we change that to something like
`Scores: forecast verification and evaluation software`